docs/computing/containers/tykky.md

```diff
@@ -130,74 +141,85 @@ Type "help", "copyright", "credits" or "license" for more information.
+An additional `--slim` flag exists, which will instead use a pre-built minimal Python
+container with a much newer version of Python as a base. Without the `--slim` flag,
+the whole host system is available, whereas with the flag the system installations (i.e.
+`/usr`, `/lib64`, ...) are no longer taken from the host, but instead coming from
+within the container.
 
-Tykky also provides an option to: 
-	
-* Generate wrappers for tools in existing Apptainer/Singularity containers, so that they can be used 
-transparently (no need to prepend `singularity exec ...`, or modify scripts if switching between containerized versions and "normal" installation).
-* Install tools available in Docker images, including generating wrappers.
+## Container-based installations
 
-```
-wrap-container -w </path/inside/container> <container> --prefix <install_dir> 
+Tykky also provides an option to:
+
+- Generate wrappers for tools in existing Apptainer/Singularity containers so that
+  they can be used transparently (no need to prepend `apptainer exec ...` or modify
+  scripts if switching between containerized versions and "normal" installations).
+- Install tools available in Docker images, including generating wrappers.
+
+```bash
+wrap-container -w /path/inside/container <container> --prefix <install_dir> 
 ```
 
-* `<container>` can be a local filepath or any [URL accepted by singularity](https://docs.sylabs.io/guides/3.7/user-guide/cli/singularity_pull.html) (e.g `docker://` `oras://` )
-* `-w` needs to be an absolute path (or comma separated list) inside the container. Wrappers will then be automatically
-created for the executables in the target directories / for the target path. If you do not know the path of executables in the container, open a shell inside the container and use [which command](https://linuxize.com/post/linux-which-command/). To open shell:
-	* In case of existing local Apptainer/Singularity file: `singularity shell xxx.sif`.
-	* In case of Docker or non-local Apptainer/Singularity file, create first the installation with some path and then start with created `_debug_shell`.
+- `<container>` can be a local filepath or any [URL accepted by
+  Apptainer/Singularity](https://docs.sylabs.io/guides/3.7/user-guide/cli/singularity_pull.html)
+  (e.g `docker://` `oras://`)
+- `-w` needs to be an absolute path (or comma-separated list) inside the container.
+  Wrappers will then be automatically created for the executables in the target
+  directories / for the target path. If you do not know the path of the executables
+  in the container, open a shell inside the container and use the [which
+  command](https://linuxize.com/post/linux-which-command/). To open a shell:
+  	- In case of existing local Apptainer/Singularity file: `singularity shell image.sif`.
+  	- In case of Docker or non-local Apptainer/Singularity file, create first the
+      installation with some path and then start with created `_debug_shell`.
 
```

Comment:
## Memory errors

With very big installations the resources available on the login-node might not be enough and Tykky might fail with a `MemoryError`. In this case the installation needs to be done on a compute node in [interactive session](../running/interactive-usage.md#sinteractive-in-puhti).

```
# Start interactive session, here with 12 Gb memory and 15 Gb local disc, increase if needed.
sinteractive -m 12000 -d 15

# Define where to store temporary files of Apptainer
export APPTAINER_CACHEDIR=$LOCAL_SCRATCH
export APPTAINER_TMPDIR=$LOCAL_SCRATCH

cd <your_installation_folder>
# Run the Tykky commands as described above
```

Reply:
@Nortamo Is this correct? Copied from Ari-Matti's RT ticket and I have used this sometimes, but I think myself did not use these export rows.. But not sure..

Reply:
Tykky appears to know to use `$LOCAL_SCRATCH` by default (if nvme is requested), so these exports shouldn't be necessary.